src/plugins/vis_default_editor/public/components/agg_params_map.ts

```diff
@@ -31,6 +31,11 @@ const buckets = {
     has_extended_bounds: controls.HasExtendedBoundsParamEditor,
     extended_bounds: controls.ExtendedBoundsParamEditor,
   },
+  [BUCKET_TYPES.IP_PREFIX]: {
+    prefixLength: controls.PrefixLengthParamEditor,
+    prefixLength64: controls.PrefixLengthParamEditor,
```

Comment:
As I said above I think one prefixLength parameter is enough here

Reply:
Updated to just have one prefixLength input box